cmd/kwild/server/build.go

```diff
@@ -622,6 +622,11 @@ func buildStatesyncer(d *coreDependencies) *statesync.StateSyncer {
 			continue
 		}
 
+		if res.Header.Height == 0 {
+			d.log.Warnf("zero height from provider %v", p)
+			continue
+		}
```

Comment:
Somewhat unrelated change here.  @charithabandi and I could not understand why sometimes a bad RPC provider would get past this build step without triggering the `failBuild` below.  My guess was that perhaps it was getting a nonsense response, but no error. :man_shrugging: wish I had checked the logs when it happened.